bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/airbase/AirbaseControlInfo.java

```diff
@@ -78,6 +93,8 @@ public Map<String, String> getParamString() {
         params.put("f_auto", fanSpeed.getAuto() ? "1" : "0");
         params.put("f_airside", fanSpeed.getAirside() ? "1" : "0");
         params.put("f_dir", Integer.toString(fanMovement.getValue()));
+        params.put("f_dir_lr", fanMovement.getValue() == 1 || fanMovement.getValue() == 3 ? "S" : "0");
+        params.put("f_dir_ud", fanMovement.getValue() == 2 || fanMovement.getValue() == 3 ? "S" : "0");
```

Comment:
Use the enum constants FanMovement.HORIZONTAL / VERTICAL / VERTICAL_AND_HORIZONTAL.

I'd also prefer if you only put `f_dir_lr/ud` when the same parameters were read from the unit, so the correct unit will receive the correct set of known parameters.

Reply:
Used the constants (way better!)
I agree that it would be better to send only the property's the device supports. On the other side, sending both should not be a problem, they should be ignored. 
Looking at a solution what do you suggest? If you or @carda21 can give me a BasicInfo parse message i can check, maybe there is some value to determine of it has combined values or not. If so i could add property `BasicInfo.hasSeperatedDirectionParams` that defaults to false.? and be set accordingly.

Otherwise it would be a manual setting or a 'not so nice' setting on first ControlInfo.parse call. wdyt?

Reply:
Inside DaikinAcUnitHandler.pollStatus() the controlInfo is populated by ControlInfo.parse(). I'd imagine adding a field inside ControlInfo to flag for separatedDirectionParams, so that when writing out controlInfo in getParamString it doesn't need to pull in another object.

Reply:
Done, please chceck if you agree with this.

Reply:
Awesome, thanks!